library/src/main/java/com/getstream/sdk/chat/adapter/MessageListItemViewHolder.java

```diff
@@ -278,6 +279,15 @@ private void configPositionsStyle() {
         }
     }
 
+    private void configUserNameAndMessageDateStyle(){
+        if (!style.isUserNameShow()){
```

Comment:
Is safer to cover else case also. It's too easy to change xml layout, forget and commit it.
```java
tv_username.setVisibility(style.isUserNameShow() ? VISIBLE : GONE)
```